kubernetes/samples/scripts/domain-lifecycle/README.md

```diff
@@ -0,0 +1,76 @@
+### Domain lifecycle sample scripts
+
+The operator provides sample scripts to start up or shut down a specific Managed Server or cluster in a deployed domain, or the entire deployed domain.
+
+**Note**: Prior to running these scripts, you must have previously created and deployed the domain.
+
+These scripts can be helpful when scripting the life cycle of a WebLogic Server domain. For information on how to start, stop, restart, and scale WebLogic Server instances in your domain, see [Domain Life Cycle](https://oracle.github.io/weblogic-kubernetes-operator/userguide/managing-domains/domain-lifecycle).
+
+#### Scripts to start and stop a Managed Server
+The `startServer.sh` script starts a Managed Server either by increasing the `spec.clusters[<cluster-name>].replicas` value for the Managed Server's cluster by `1` or by updating the `spec.managedServers[<server-name>].serverStartPolicy` attribute of the domain resource or both as necessary. The script provides an option to keep the `spec.clusters[<cluster-name>].replicas` value constant for clustered servers. See the script `usage` information by using the `-h` option.
+
+Use the following command to start the server along with increased replica count:
```

Comment:
start the server along with increased replica count -> start the server along with an increased replica count OR start the server with an increased replica count

Reply:
Thanks, I have reworded it to say "start the server either by increasing the replica count or by updating the server start policy".